flycheck.el

```diff
@@ -5733,7 +5733,22 @@ https://github.com/rust-lang/rust/blob/master/src/libsyntax/json.rs#L67-L139"
               :checker checker
               :buffer buffer
               :filename primary-filename)
-             errors)))))
+             errors)))
+
+        ;; If there are no spans, the error is not associated with a specific
+        ;; file but with the project as a whole.  We still need to report it to
+        ;; the user by emitting a corresponding flycheck-error object.
+        (unless spans
+          (push (flycheck-error-new-at
+                 ;; We have no specific position to attach the error to, so
+                 ;; let's use the top of the file.
+                 0 0
```

Comment:
You're right, it should be 1 per the docs.  Though, it seems flycheck will treat a 0 as if the column was not specified and highlight the whole line in the `columns` highlighting mode.

Same goes for the line argument: 0 or 1 will highlight the same line.

If I introduce a parse error on the first character of the file, the error position reported by the compiler will be (1,1), so we can use that for consistency.